test/lib/type/string_test.exs

```diff
@@ -5,8 +5,10 @@ defmodule Antikythera.StringTypesTest do
   alias Antikythera.{VersionStr, Domain, EncodedPath, UnencodedPath, Email, Url, ContextId, TenantId}
 
   test "validate VersionStr" do
-    v = Mix.Project.config()[:version]
-    assert VersionStr.valid?(v)
+    assert VersionStr.valid?("0.0.1-20180501235959+0123456789abcdef0123456789abcdef01234567")
+
+    # multi-digit numbers in semantic version part are currently not allowed
```

Comment:
The whole version string can be seen as a "semantic version part".
We should write something like
"multi-digit numbers in major/minor/patch part are currently not allowed"

Reply:
Fixed in d2bd470
Also showing a valid `Antikythera.VersionStr` string is parsable by Elixir's `Version` too.